terraform/test-environments/modules/chef_automate_install/templates/install_chef_automate_cli.sh.tpl

```diff
@@ -124,6 +124,11 @@ configure_retention() {
   }'
 }
 
+populate_integration_data_data() {
+  hab pkg install core/go -b
+  TOTAL_RECORDS=100 INFRA_SERVER_ID="auto-deployed-chef-server" INFRA_SERVER_ORG_ID="auto-deployed-chef-org" A2_SVC_NAME="infra-proxy-service" A2_SVC_PATH="/hab/svc/infra-proxy-service" go test "github.com/chef/automate/components/infra-proxy-service/integration_test"
+}
+
```

Comment:
@jeremiahsnapp please reviews these changes? Thanks!

Reply:
Looks good to me